infinario.py

```diff
@@ -278,12 +258,23 @@ def __init__(self, token, customer=None, target=None, silent=True, logger=None,
         :param logger: Instance of a logger used with silent == True
         :param transport: One of `NullTransport`, `SynchronousTransport`, `AsynchronousTransport`;
             consult their documentation as well
+        :param secret: Secret token of a project with analyses that should be exported
```

Comment:
Maybe it should be noted here that it is optional and that it can be used also for segment determination not just analysis exports.